model/src/main/proto/exploration_checkpoint.proto

```diff
@@ -30,24 +30,17 @@ message ExplorationCheckpoint {
   // The index of the current selected state from the deck of states.
   int32 state_index = 4;
 
-  // Stores -1 if no hint for the pending top state has been revealed otherwise it stores the index
-  // of the hint that the learner has revealed.
-  int32 hint_index = 5;
-
-  // Stores true if the solution to the pending top state has been revealed by the learner otherwise
-  // it stores false.
-  bool solution_is_revealed = 6;
-
   // The title of the exploration whose checkpoint is saved.
-  string exploration_title = 7;
+  string exploration_title = 5;
```

Comment:
Avoid modifying existing proto fields. Instead you can deprecate fields 5 and 6 and keep 7 as is.
i.e. 
```
reserved 5,6;

// ....
string exploration_title = 7;

// ...
string exploration_version = 8;

.....
HelpIndex help_index = 10;
```

Reply:
done